### ISSUE TITLE
Backslash prepended to uploaded file name

### ISSUE THREAD

**Issue by miguelsm** [MikaAK/s3-plugin-webpack#27]
After upgrading to 0.6.7 I've noticed the file name is being prepended a backslash. Up to 0.6.6 the uploaded file was correctly named in my bucket as `bundle.js`, whereas now the uploaded file name reads `\bundle.js`.

This is how I've got the plugin configured in my `webpack.config.js`:

```javascript
new S3Plugin({
  include: /bundle.js/,
  s3Options: {
    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
    region: 'us-east-1'
  },
  s3UploadOptions: {
    Bucket: 'dev.bucket'
  },
  basePath: 'some/path/dev',
  directory: path.join(__dirname, 'dist')
})
```

I'm running node v5.8 in Windows 7.

**Comment by MikaAK** [comment]
I cannot replicate this. @miguelsm can you try again with the latest version and tell me if it still occurs?

**Comment by MikaAK** [comment]
closing since no response from OP. Feel free to reopen.

**Comment by ytanay** [comment]
I have also seen this issue occurring on Windows - I've opened a PR with a possible fix (#36).
